### ISSUE TITLE
TeamTalk 5 default QT client, (accessibility bug report): the exact list information are not announced by screen reader

### ISSUE THREAD

**Issue by shaghayegh1382** [BearWare/TeamTalk5#1529]
### Description

Exact list information for channels and users are not announced by screen readers, it just says: level 1, level 2, for all users and channels.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [X] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

I hope exact list information will be announced in the next version. Assuming 3 users. currently, when navigating between users and channel my screen reader says as follow:
"albatros", level 1. "Space cat", level 1, "Bioman", level 1. 
It must say: "Albatros", 1 of 3 level 1, "Space cat", 2 of 3 level 1, "Bioman", 3 of 3 level 1

### Actual behavior

I navigate between users and channels but lisf info is not exactly announced

### Steps to reproduce problem

step 1: I enter to a server.
step 2: I navigate through channels and users.
step 3: for example, if there are 3 users in root channel, I navigate between them.
step 4: When navigating, assuming 3 users, it says: "albatros", level 1. "Space cat", level 1, "Bioman", level 1.

**Comment by amirmahdifard** [comment]
it's doesn't matter if it's not saying 1 of 3: but does it says the user's gender and they status if they have a status? or not

**Comment by amirmahdifard** [comment]
i don't have this problem because all needed information is announced for me.

**Comment by CoBC** [comment]
Related to Qt issue, reported here:
[QTBUG-104569](https://bugreports.qt.io/browse/QTBUG-104569)

**Comment by amirmahdifard** [comment]
wy this is closed suddenly?
